cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -45,11 +49,40 @@ const Modules = () => {
         setLoadingModules(false);
       });
   }, []);
+  useEffect(() => {
+    setFilteredData(
+      filteredData.filter((module: any) =>
+        moduleHealthFilter
+          .map((status) => status.toLowerCase())
+          .includes(module.status.toLowerCase()),
+      ),
+    );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [moduleHealthFilter]);
```

Comment:
@nikhiltanna, lets add the missing dependency instead of silencing it

Reply:
If we add it will keep on retriggering and will set the infinite loop of renders

Reply:
Have updated the dependency array as well sorry for the issue